src/Compression/Enums.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+using Foundation;
+using CoreFoundation;
+using ObjCRuntime;
+using CoreVideo;
+
+namespace Compression {
+
+	public enum CompressionAlgorithm {
+		LZ4 = 0x100,
+		LZ4Raw = 0x101,
+		Lzfse = 0x801,
+		Lzma = 0x306,
+		Zlib = 0x205,
+	}
+
+	public enum CompressionStatus {
+		Ok = 0,
+		End = 1,
+		Error = -1,
+	}
+
+	[Flags]
+	public enum StreamFlag {
```

Comment:
from what I can see this enum should not be public, it's not exposed (and .net fxdg does not like `Flag` suffixes)